Pet-Clinic-Frontend/src/pages/Login.tsx

```diff
@@ -1,51 +1,87 @@
 import { useState } from 'react'
-import './Login.css'
+import { useNavigate } from 'react-router-dom';  // must be install by command: npm list react-router-dom
```

Comment:
It should be installed from the `package.json` already, should not install it manually after that.